.github/workflows/build-and-install.yml

```diff
@@ -25,9 +26,9 @@ jobs:
           - 'fedora:32'
           - 'fedora:31'
           - 'fedora:30'
-          - 'opensuse/leap:15.2'
-          - 'opensuse/leap:15.1'
-          - 'opensuse/tumbleweed:latest'
+            #- 'opensuse/leap:15.2'
+            #- 'opensuse/leap:15.1'
+            #- 'opensuse/tumbleweed:latest'
```

Comment:
If we're just disabling fail-fast behavior, we can probably safely keep these enabled.

Reply:
See my previous comments about this in other PRs. I'd rather we remove openSUSE entirely for the time being.